packages/shared/src/lib/featureValues.ts

```diff
@@ -29,3 +31,7 @@ export enum SearchExperiment {
   Control = 'control',
   V1 = 'v1',
 }
+
+const anonymousMigrationDate = new Date(2023, 10, 4);
```

Comment:
I'm just asking product for the exact date, unless you got it already from them?

Reply:
Asked Nimrod and last day available should be:
So October 31st, 23:59 would be a good deadline I guess

So you would set 1st of November basically :) 
Let's do that and the rel and ship it 🚀

Reply:
Nice. Thanks for checking with them. Should be updated now 🚢